EIPS/eip-3157.md

```diff
@@ -0,0 +1,265 @@
+---
+eip: 3157
+title: Batch Flash Loans
+author: Alberto Cuesta Cañada (@albertocuestacanada), Fiona Kobayashi (@fifikobayashi), fubuloubu (@fubuloubu), Austin Williams (@onewayfunction)
+discussions-to: https://ethereum-magicians.org/t/erc-3157-candidate-batch-flash-loans/5260
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-01-31
+---
+
+## Simple Summary
+
+This ERC provides standard interfaces and processes for flash lenders and borrowers, allowing for flash loan integration without a need to consider each particular implementation. The flash loans covered by this ERC can be composed of multiple assets at once, as opposed to ERC3156 where a single asset is included in each loan.
```

Comment:
This is a bit long for a simple summary.  Think of a GitHub PR title, email subject line, forum thread title, etc.  Consider moving some or all of this to the abstract and having the simple summary just be a nice one-line statement.

Reply:
Fixed, will do the same for ERC3156.